packages/hooks/src/useElementNode.ts

```diff
@@ -0,0 +1,25 @@
+import { useState, useCallback } from 'react';
+
+/**
+ * Hook to subscribe to changes in a ref element.
+ * Example:
+ * ```
+ * const Example = () => {
```

Comment:
I think examples you don't need to include the `*`. Hover over the function to see how it's rendered.

Reply:
👍